src/user-management/user-management.ts

```diff
@@ -961,6 +961,23 @@ export class UserManagement {
     return `${this.workos.baseURL}/user_management/sessions/logout?session_id=${sessionId}`;
   }
 
+  async getLogoutUrlFromSessionCookie({
+    sessionData,
+    cookiePassword = process.env.WORKOS_COOKIE_PASSWORD,
+  }: SessionHandlerOptions): Promise<string> {
+    const authenticationResponse = await this.authenticateWithSessionCookie({
+      sessionData,
+      cookiePassword,
+    });
+
+    if (!authenticationResponse.authenticated) {
+      const { reason } = authenticationResponse;
+      throw new Error(`Failed to authenticate session: ${reason}`);
```

Comment:
This error is weird. I'd be surprised and confused if I tried to get a log out URL and was given a "failed to authenticate" error. How about 

```suggestion
      throw new Error(`Failed to extract session ID for logout URL: ${reason}`);
```

Reply:
Makes sense, updating!